templates/commands/goldentest/new_test_cli.go

```diff
@@ -142,24 +144,20 @@ func (c *NewTestCommand) Run(ctx context.Context, args []string) (rErr error) {
 }
 
 func marshalTestCase(inputs, builtinVars map[string]string) ([]byte, error) {
```

Comment:
Can you add a test case to make sure the simple header works, see example here  https://github.com/abcxyz/abc/pull/386/files#diff-a3e6090ddcab7dc561a353a6ec6d5df1847d1452887fffda4c00c29cf93c815fR121

Reply:
Copied yours, thanks.